core/storage/translation/gae_models.py

```diff
@@ -34,6 +35,160 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+# TODO(#14537): This is a duplicate of the same TypedDict in translation_domain,
+# it should be removed after we refactor our importing.
+class TranslatedContentDict(TypedDict):
+    """Dictionary representing TranslatedContent object."""
+
+    content: str|List[str]
+    needs_update: bool
+
+
+class EntityTranslationsModel(base_models.BaseModel):
+    """Model for storing entity translations ."""
+
+    # The id of the corresponding entity.
+    entity_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The type of the corresponding entity.
+    entity_type = datastore_services.StringProperty(required=True, indexed=True)
+    # The version of the corresponding entity.
+    entity_version = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The ISO 639-1 code for the language an entity is written in.
+    language_code = datastore_services.StringProperty(
+        required=True, indexed=True)
+    # The translated content.
```

Comment:
This is a JSON property, so explain its internal format in detail.

Reply:
Updated.